### PR TITLE
change workflow to allow actionner to also delete pods

### PR DESCRIPTION
**Pull request by IgorEulalio**
https://github.com/falco-talon/falco-talon/issues/451

### CODE REVIEW

File: actionners/kubernetes/terminate/terminate.go [hunk 1]
```diff
@@ -50,6 +50,7 @@ rules:
     grace_period_seconds: 5
     ignore_daemonsets: true
     ignore_statefulsets: true
+	ignore_pods: false
```
Comment (Issif) [inline actionners/kubernetes/terminate/terminate.go:53]:
```suggestion
  ignore_pods: false
```

for consistency, set to "true" all settings, I prefer `delete_standalone_pods`. It also mean, the default value is `false` like it is right now

**Review by Issif — CHANGES_REQUESTED** [review]

File: actionners/kubernetes/terminate/terminate.go (hunk 1 above)
Comment (IgorEulalio) [reply actionners/kubernetes/terminate/terminate.go:53]: I started with delete_standalone_pods, but for consistency I decided to change it to ignore_pods.

I can set it to true, but are you sure changing it here has any effect on the real app your you're meaning more from the documentation perspective?

Comment (IgorEulalio) [reply actionners/kubernetes/terminate/terminate.go]: Changed it to true.

File: actionners/kubernetes/terminate/terminate.go [hunk 2]
```diff
@@ -139,13 +142,7 @@ func (a Actionner) Run(event *events.Event, action *rules.Action) (utils.LogLine
...
 	ownerKind, err := k8s.GetOwnerKind(*pod)
 	if err != nil {
-		return utils.LogLine{
-				Objects: objects,
-				Error:   err.Error(),
-				Status:  utils.FailureStr,
-			},
-			nil,
-			err
+		ownerKind = utils.StandalonePodStr
```
Comment (Issif) [inline actionners/kubernetes/terminate/terminate.go:145]: are you sure we get an error and not a nil error + a kind as `Standalone`

**Review by Issif — CHANGES_REQUESTED** [review]

**Review by Issif — APPROVED** [review]

**Pull request merged**